Packet++/src/TcpReassembly.cpp

```diff
@@ -800,4 +810,38 @@ int TcpReassembly::isConnectionOpen(const ConnectionData& connection) const
 	return -1;
 }
 
+void TcpReassembly::insertIntoCleanupList(uint32_t flowKey, time_t currentTime)
+{
+	std::pair<CleanupList::iterator, bool> pair = m_CleanupList.insert(std::make_pair(currentTime + m_ClosedConnectionDelay, CleanupList::mapped_type()));
+	CleanupList::mapped_type &keysList = pair.first->second;
+	keysList.push_back(flowKey);
+}
+
+void TcpReassembly::purgeClosedConnections(time_t currentTime, uint32_t maxNumToClean)
+{
+	uint32_t count = 0;

```

Comment:
this method implementation is pretty complex in its nature. It'd beneficial if you can add comments to make it easier for people going over the code to understand its flow

Reply:
Comments were added